flask_sqlalchemy/__init__.py

```diff
@@ -403,6 +403,11 @@ def iter_pages(self, left_edge=2, left_current=2,
                 yield num
                 last = num
 
+    def as_dict(self):
+        """Returns dict representation of the object"""
+        return dict(items=self.items, page=self.page, total=self.total,
+                    prev_num=self.prev_num, next_num=self.next_num)
```

Comment:
Use a dict literal here, not the callable, and use single-indent continuation style:

~~~python
return {
    'items': items,
    ...
}
~~~